db/schema.rb

```diff
@@ -607,15 +607,17 @@
   add_index "schedules", ["facility_id"], name: "i_schedules_facility_id", using: :btree
 
   create_table "secure_rooms_card_readers", force: :cascade do |t|
-    t.integer  "product_id",            limit: 4,   null: false
+    t.integer  "product_id",            limit: 4,                  null: false
     t.string   "card_reader_number",    limit: 255
     t.string   "control_device_number", limit: 255
-    t.datetime "created_at",                        null: false
-    t.datetime "updated_at",                        null: false
+    t.datetime "created_at",                                       null: false
+    t.datetime "updated_at",                                       null: false
     t.string   "description",           limit: 255
+    t.boolean  "direction_in",                      default: true, null: false
```

Comment:
We default to `true`, and there's no way to set it to anything but true/false in the interface, so if we hit the db null constraint, it means someone's up to no good. Basically, I don't feel like a validation is necessary.